Makefile

```diff
@@ -188,7 +198,10 @@ push-plugin: plugin-container
 push-datamgr: datamgr-container
 	docker push $(DATAMGR_IMAGE):$(VERSION)
 
-push: push-datamgr push-plugin
+push-backupdriver: backupdriver-container
```

Comment:
push-backup-driver, backup-driver-container
I know, we should probably make it data-manager-container too